com.sublimemerge.App.yaml

```diff
@@ -58,4 +59,15 @@ modules:
       - install apply_extra /app/bin
       - install sublime_merge.sh /app/bin/sublime_merge
       - install -Dm644 com.sublimemerge.App.appdata.xml /app/share/appdata/com.sublimemerge.App.appdata.xml
-      - install -Dm644 com.sublimemerge.App.png /app/share/icons/hicolor/128x128/apps/com.sublimemerge.App.png
\ No newline at end of file
+      - install -Dm644 com.sublimemerge.App.png /app/share/icons/hicolor/128x128/apps/com.sublimemerge.App.png
+  - name: git-lfs
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/libexec/git-lfs/
+      - install -Dm755 git-lfs /app/libexec/git-lfs/
+      - sed -i '2i/app/libexec/git-lfs/git-lfs install --system >/dev/null' /app/bin/sublime_merge
```

Comment:
You can probably just put this line directly in the `sublime_merge.sh` script where it's created above (currently lines 41-44)